others_recipes/atamasoup.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Atama Soup</h1>
+    <img src="whitesoup_images/atama soup.jpg" />
```

Comment:
- [ ] Please always ensure that your `img` tag has an `alt` attribute to enable users to know what image should display in the event that the image did not appear on the page as expected. Read more [about it here](https://www.w3schools.com/tags/att_img_alt.asp#:~:text=The%20required%20alt%20attribute%20specifies,user%20uses%20a%20screen%20reader).)